contracts/gauges/gauge/src/msg.rs

```diff
@@ -94,37 +97,46 @@ pub enum ExecuteMsg {
 pub struct CreateGaugeReply {
     /// Id of the gauge that was just created
     pub id: u64,
+    pub addr: String,
 }
 
 /// Queries the gauge exposes
 #[cw_serde]
```

Comment:
Can we implement the standard `cw_ownable` queries? There are many examples for how to do this in `dao-contracts`.